rct229/rulesets/ashrae9012019/section16/section16rule7.py

```diff
@@ -0,0 +1,64 @@
+from rct229.schema.config import ureg
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_description
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+from rct229.utils.pint_utils import ZERO
+from rct229.utils.pint_utils import CalcQ
+
+REQ_ELEVATOR_CAB_LIGHTING_POWER_DENSITY = 3.14 * ureg("W/ft2")
+
+
+class Section16Rule7(RuleDefinitionListIndexedBase):
+    """Rule 7 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule7, self).__init__(
+            rmds_used=produce_ruleset_model_description(
+                USER=False, BASELINE_0=True, PROPOSED=True
+            ),
+            each_rule=Section16Rule7.ElevatorRule(),
+            index_rmd=BASELINE_0,
+            id="16-7",
+            description="When included in the proposed design, the baseline elevator cab lighting power density shall be 3.14 W/ft2.",
+            ruleset_section_title="Elevators",
+            standard_section="Table G3.1-16 Baseline Building Performance",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+        return find_all(
```

Comment:
Could you follow https://github.com/pnnl/ruleset-checking-tool/pull/1342#:~:text=%40JacksonJ%2DKC%20%2C%20could%20you%20check%20what%20outcome%20should%20it%20be%20if%20the%20baseline%20has%20no%20elevator%20but%20proposed%20has%3F%0AThe%20rule%20passes%20if%20no%20elevator%20in%20baseline%20model%20but%20proposed%20has. and fix this is applicable logic?